Update links

KurogeWashu opened the pull request:
Update links in smart contract section

KurogeWashu committed:
Updated smart contract links

saltyfacu commented on developers/code-repositories.md:
```diff
@@ -17,13 +17,13 @@
 
 ## Smart Contracts
 
-Yearn smart contracts repo: [https://github.com/yearn/yearn-protocol](https://github.com/yearn/yearn-vaults)
```
yearn-vaults is the correct repo. The description needs to change.

yearn-protocol is for v1 vaults

KurogeWashu replied:
- [x]

saltyfacu commented on developers/code-repositories.md:
```diff
@@ -17,13 +17,13 @@
-- [Strategies](https://github.com/yearn/brownie-strategy-mix)
-- [Token Wrappers](https://github.com/yearn/brownie-wrapper-mix)
+- [Vaults](https://github.com/yearn/yearn-vaults)
+- [Strategies](https://github.com/yearn/yearn-protocol/tree/develop/contracts/strategies)
```
strategies are in separate repos, this should be removed

KurogeWashu replied:
Should we keep the controllers, registries, and utils links from yearn-protocol?

saltyfacu replied:
it's really not used anymore :/

The registry is all v2 vaults. But that repo is deprecated for good

saltyfacu commented on developers/code-repositories.md:
```diff
@@ -17,13 +17,13 @@
-- [Token Wrappers](https://github.com/yearn/brownie-wrapper-mix)
+- [Vaults](https://github.com/yearn/yearn-vaults)
+- [Strategies](https://github.com/yearn/yearn-protocol/tree/develop/contracts/strategies)
+- [Token Wrappers](https://github.com/yearn/yearn-vaults/blob/main/contracts/BaseWrapper.sol)
```
here we should add base router also: https://github.com/yearn/yearn-vaults/blob/main/contracts/BaseRouter.sol

KurogeWashu replied:
- [x]

KurogeWashu committed:
Revisions

- updated smart contracts repo link: to yearn-vaults from yearn-protocol (v1)
- removed strategies
- added base wrapper

saltyfacu reviewed (approved):
lgtm
